### ISSUE TITLE
Double click effect that should not appear

### ISSUE THREAD

**Issue by BalrogBlade** [linuxmint/nemo#2198]
```
 * Nemo version 4.2.2
 * all
 * Distribution  linuxmint 19.2, 
 * 64 bit
 ```

**Issue**
In some special cases, a quick right-click after clicking the left mouse button on the catalog will have a double-click effect.
Originally expected to open the right-click menu, the result is to execute the open file or directory

**Steps to reproduce**
1.Click the left mouse button
2.Quickly click the right mouse button

**Expected behaviour**

**Other information**

**Comment by shadow73s** [comment]
I can confirm this Bug.

Nemo 4.2.3
Linux Mint 19.2
64bit

**Comment by xenopeek** [comment]
There are two parts to this. First, in mouse settings by default middle-click emulation is enabled: clicking left and right button at (almost) the same time will emulate middle-click. If you have a middle button and you're a very fast clicker, you may want to disable that to avoid erroneous middle-clicks.

![image](https://user-images.githubusercontent.com/21159972/85967397-a5856c00-b9c2-11ea-970e-57611e997e9d.png)

Second part is that Nemo has an action tied to middle-click namely folders you middle-click will be opened in a new tab. Quite handy. For files it does the same as double-clicking.

In short, probably disabling middle-click emulation in mouse settings will solve this for you.
